gobblin-metrics/src/test/java/gobblin/metrics/influxdb/InfluxDBReporterTest.java

```diff
@@ -177,7 +167,7 @@ public void tearDown() {
    */
   private static class TestInfluxDB extends InfluxDBImpl {
```

Comment:
No, I am not aware of any either. influxdb-java tested the only implementation `InfluxDBImpl` using a Docker image of InfluxDB.